src/main/scala/akka/persistence/cassandra/journal/CassandraJournal.scala

```diff
@@ -56,14 +64,23 @@ class CassandraJournal(cfg: Config) extends AsyncWriteJournal with CassandraReco
   def asyncWriteMessages(messages: Seq[AtomicWrite]): Future[Seq[Try[Unit]]] = {
     // we need to preserve the order / size of this sequence even though we don't map
     // AtomicWrites 1:1 with a C* insert
-    val serialized = messages.map(aw => Try {
+    // We must NOT catch serialization exceptions here because rejections will cause
+    // holes in the sequence number series and we use the sequence numbers to detect
+    // missing (delayed) events in the eventByTag query
```

Comment:
Is that the correct approach though? I thought the AsyncWriteJournal would handle the failures appropriately. How do we now signal issues and how does the caller cope with it?